tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java

```diff
@@ -0,0 +1,254 @@
+		 * was changed and gives credit to the translator.
+		 * 
+		 * You don't need to use the line above, and you don't have have to give
+		 * yourself credit, but you should show that the language has changed.
+		 * For example, in the German translation, I just used the line
+		 * "Nichts leichter als das!", which translates literally to
+		 * "Nothing easier than that!", which refers to German being my first
+		 * language.
+		 * 
+		 * Tillerino
+		 * 
+		 * P.S. you can put a link to your profile into the line like this:
+		 * [https://osu.ppy.sh/u/2070907 Tillerino]
+		 */
+		return new Message("_BibleThump hjalp meg til å lære norsk.");
+	}
+
+	@Override
+	public String invalidChoice(String invalid, String choices) {
+		return "Beklager, men \"" + invalid
+				+ "\" kan ikke beregnes. Prøv disse: " + choices + "!";
+	}
+
+	@Override
+	public String setFormat() {
+		return "Syntaksen for å sette parameter er !set option value. Prøv !help hvis du trenger mer veiledning.";
+	}
+	
+	StringShuffler doSomething = new StringShuffler(rnd);
+	
+	@Override
+	public String apiTimeoutException() {
+		final String message = "osu! serverene er ikke så raske akkurat nå, så det er ingenting jeg kan gjøre for deg i dette øyeblikket. ";
+		return message + doSomething.get(
+				"Fortell... Når sist snakket du med bestemoren din?",
+				"Hva med at du rydder rommet ditt så spør du meg igjen?",
+				"Jeg vedder på at du ville elsket å gått en tur akkuratt nå. Du vet... ut?",
+				"Jeg bare vet at du har utrolig mye andre ting å gjøre. Hva med at du gjør dem nå?",
+				"Det ser du som du trenger litt søvn uansett.",
+				"Men skjekk ut denne super interresange siden på [https://en.wikipedia.org/wiki/Special:Random wikipedia]!",
+				"La oss se om det er noen gode spillere som [http://www.twitch.tv/directory/game/Osu! streamer] akkurat nå!",
+				"Se, her er et annet [http://dagobah.net/flash/Cursor_Invisible.swf spill] som du mest sansynlig kommer til å være dårlig i!",
+				"Dette bør gi deg mye tid til å studere [https://github.com/Tillerino/Tillerinobot/wiki manualen min].",
+				"Ikke beskymre deg! Disse [https://www.reddit.com/r/osugame danke memsene] bør få tiden til å fly.",
+				"Mens du kjeder deg, gi [http://gabrielecirulli.github.io/2048/ 2048] et forsøk!",
+				"Morsomt spørsmål: Hvis harddisken din krasjet akkurat nå, hvor mye av din personlige data vil bli tapt for altid?",
+				"Så... Har du noen gang prøvd [https://www.google.de/search?q=bring%20sally%20up%20push%20up%20challenge sally up push up challenge]?",
+				"Du kan gjøre noe annet eller vi kan stirre inn i øynene til hverandre mens vi er stille."
+				);
+	}
+
+	@Override
+	public String noRecentPlays() {
+		return "Jeg har ikke sett deg spille i det siste.";
+	}
+	
+	@Override
+	public String isSetId() {
+		return "Denne refererer til et set med maps ikke en enkelt map.";
+	}
+	
+	@Override
+	public String getPatience() {
+		return "Et øyeblikk.";
+	}
+} //end of file
```

Comment:
nuuu, i didnt mean this :(
then again, I probably shouldnt be complaining about this, but what I meant, was just press the enter key once in your editor so it has a new empty line at the end of the file